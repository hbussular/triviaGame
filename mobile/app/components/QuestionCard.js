import React from 'react'
import PropTypes from 'prop-types'
import { View, Text } from 'react-native'
import styles from './QuestionCard.style'

const CardQuestion = ({ question }) => (
  <View style={styles.viewContainer}>
    <Text style={styles.textContainer}>
      {question}
    </Text>
  </View>
)

CardQuestion.propTypes = {
  question: PropTypes.string.isRequired,
}

export default CardQuestion
