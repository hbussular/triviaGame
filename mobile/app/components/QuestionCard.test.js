import React from 'react'
import renderer from 'react-test-renderer'
import QuestionCard from './QuestionCard'

import { questionScenario } from '../tests/scenarios'

test('renders correctly', () => {
  const props = { question: questionScenario.question }
  const tree = renderer
    .create(<QuestionCard {...props} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
