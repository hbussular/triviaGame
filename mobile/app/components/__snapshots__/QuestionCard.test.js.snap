// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 8,
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "padding": 30,
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 16,
        "textAlign": "center",
      }
    }
  >
    Which &quot;Toy Story&quot; character was voiced by Don Rickles?
  </Text>
</View>
`;
